src/utils/nativeTokens.ts

```diff
@@ -12,6 +12,6 @@ export function getNativeTokenDBAddress(chain: Chain): string {
     case Chain.Arbitrum:
     case Chain.EthereumGoerli:
     case Chain.Optimism:
-      return 'ETH'
+      return undefined
```

Comment:
i would just return `null` here so you don't need to override in the actual query